core/src/main/scala/kafka/log/LogConfig.scala

```diff
@@ -206,18 +214,22 @@ object LogConfig {
       this
     }
 
-    override def headers = List("Name", "Description", "Type", "Default", "Valid Values", "Server Default Property", "Importance").asJava
+    override def headers = List("Name", "Description", "Type", "Default", "Valid Values", ServerDefaultHeaderName,
+      "Importance").asJava
 
     override def getConfigValue(key: ConfigKey, headerName: String): String = {
```

Comment:
I wonder if we can come up with a better name. I was having a hard time understanding this patch because this is named so generically. Maybe something like `docStringForHeader` or  something like that?

Reply:
Yeah, I agree that it's not a good name. I thought about renaming it too, but then decided to limit the scope. Happy to do it given your feedback.

Reply:
I guess this is grey area as far as compatibility is concerned since `ConfigKey` is public. Let's go ahead and leave it for now.

Reply:
`getConfigValue` is defined in `ConfigDef` and it's `protected`. However, `ConfigDef` is designed to be inherited from, so it could be considered a compatibility break to rename it. I agree with you that it's best to leave it for now.